server/src/routes/api/v1/auth/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Router, Request, Response, NextFunction } from "express";
```

Comment:
Yes, we could rename the routes files. This is a common practice on node. I add this to the rename issue.